eng/pipelines/runtime.yml

```diff
@@ -764,6 +764,37 @@ jobs:
     jobParameters:
       isOfficialBuild: false
       isFullMatrix: ${{ variables.isFullMatrix }}
+      runtimeDisplayName: mono
+      testScope: innerloop
+      liveRuntimeBuildConfig: release
+      dependsOnTestBuildConfiguration: ${{ variables.debugOnPrReleaseOnRolling }}
+      dependsOnTestArchitecture: x64
+      condition: >-
+        or(
+          eq(dependencies.checkout.outputs['SetPathVars_libraries.containsChange'], true),
+          eq(dependencies.checkout.outputs['SetPathVars_mono.containsChange'], true),
+          eq(variables['isFullMatrix'], true))
+
+#
+# Libraries Release Test Execution against a release mono interpreter runtime.
+# Only when libraries or mono changed
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/libraries/run-test-job.yml
+    runtimeFlavor: mono
+    buildConfig: ${{ variables.debugOnPrReleaseOnRolling }}
+    platforms:
+    # - Windows_NT_x64
+    #- OSX_x64
+    #- Linux_arm64
+    - Linux_x64
```

Comment:
It makes me wonder if we should add a new platform-matrix and consume more resources by adding more legs. Should we instead introduce a new multi-helix-job project that calls into `sendtohelix.proj` multiple times with different variables in parallel? 

Just throwing some ideas here, that will also simplify the build whenever we move mono to build and run tests as part of the same build leg, so that way we wouldn't need to build the tests twice, etc.

Reply:
I think there are efficiencies to be had for sure.  It would be nice if we could run multiple jobs like you describe and still maintain a 'separate leg' view in the UI.  I feel it's important to have the interpreter be first class / distinct in that regard.  

We definitely should discuss / throw out more ideas.

Reply:
> jobs like you describe and still maintain a 'separate leg' view in the UI

I don't think that would be possible, but I think I wouldn't mind having one leg for the mono tests and then having multiple test modes in that single run leg. our infra should be resilient to display the test results correctly for that. 

Runtime tests do that, they send regular runs and non tiered compilation runs I believe.

Reply:
Discussed offline and @steveisok is going to open a follow up PR to address this.

Reply:
What is the tracking issue? We should add it to the change here for tracking.

Reply:
https://github.com/dotnet/runtime/issues/36215